### DISCUSSION TITLE
Replace pipeline element recommender with better search function

### DISCUSSION THREAD

**Discussion by dominikriemer** [apache/streampipes#3187] [Dev]
Hi,
in the pipeline editor, we have a `Recommended elements` feature which shows "recommended" elements based on previous connections (screenshot below).
I think the feature has only limited value (since there is already a filter for valid pipeline elements) and it requires to store all connections in a dedicated database. I suggest to remove this feature and replace it with a better search mechanism for pipeline elements (e.g. supporting full-text search for a specific pipeline element based on the documentation).

Does anyone think we should keep the recommender feature?

![image](https://github.com/user-attachments/assets/a41098d5-901c-48f6-868e-257a9e06514d)

**Comment by tenthe** [comment]
Yes, I fully agree. In my opinion we can remove it. As a next step we can provide a better search capability for processing elements.
